### DISCUSSION TITLE
How to make GCR Organization Image Public?

### DISCUSSION THREAD

**Discussion by wei** [community/community#26014] [Actions]
<p>As titled, we successfully pushed to GCR, however I’m not sure how to make the package public. Could someone show me where to toggle it to public? Thank you!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/240f9a4a45c5f974171789de86b6cb4ffdc7c17b.png" data-download-href="/uploads/short-url/590ChTj34gmfIMglLK80qbPfYfh.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5880908/181094763-70d8e825-9a91-4ced-a396-0311f7fa3491.png" alt="image" data-base62-sha1="590ChTj34gmfIMglLK80qbPfYfh" width="690" height="422" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/240f9a4a45c5f974171789de86b6cb4ffdc7c17b_2_690x422.png, https://aws1.discourse-cdn.com/github/original/2X/2/240f9a4a45c5f974171789de86b6cb4ffdc7c17b.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/240f9a4a45c5f974171789de86b6cb4ffdc7c17b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/240f9a4a45c5f974171789de86b6cb4ffdc7c17b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">994×608 37.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/12522076b4388b98fca4ef04d2d52efa7ce7dbdf.png" data-download-href="/uploads/short-url/2C4y7xKNNJvHk9SQvK4EfVJa9LV.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5880908/181094766-ad7d536f-b1d9-452e-b328-17684394be57.png" alt="image" data-base62-sha1="2C4y7xKNNJvHk9SQvK4EfVJa9LV" width="690" height="482" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/12522076b4388b98fca4ef04d2d52efa7ce7dbdf_2_690x482.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/12522076b4388b98fca4ef04d2d52efa7ce7dbdf_2_1035x723.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/12522076b4388b98fca4ef04d2d52efa7ce7dbdf_2_1380x964.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/12522076b4388b98fca4ef04d2d52efa7ce7dbdf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2002×1400 184 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/017416ae6f684de2db2e01b285649cedfaf76a47.png" data-download-href="/uploads/short-url/cRc9ZlkNf1EiDI76mHflBfOUwD.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5880908/181094767-afcae248-6541-4fb5-8178-1e6da4aa644a.png" alt="image" data-base62-sha1="cRc9ZlkNf1EiDI76mHflBfOUwD" width="616" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/017416ae6f684de2db2e01b285649cedfaf76a47_2_616x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/017416ae6f684de2db2e01b285649cedfaf76a47_2_924x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/017416ae6f684de2db2e01b285649cedfaf76a47_2_1232x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/017416ae6f684de2db2e01b285649cedfaf76a47_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2026×1642 221 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by noahmatisoff** [comment, marked as answer]
<p>Hi <a class="mention" href="/wei">@wei</a>! The option to make the container public isn’t visible because the setting must be enabled on the organization. Go to the organization settings -&gt; member privileges -&gt; package creation, and then enable public packages.</p>
<p>After that, you should see the option to make it public there in the danger zone. :smiley:</p>

**Reply by Olshansk** [reply]
This is still the correct answer as of September 2024.

**Comment by wei** [comment]
<p>Thank you for the help That was it! I was able to publish the package (as an org owner) without enabling package creation permissions in settings. It’s a little confusing but I got it!</p>

**Comment by alexellis** [comment]
<p>I ran into the same issue, and there is no obvious way of finding the way to fix the issue. The button should at least be present but disabled with a hint as to why.</p>

**Comment by clarkbw** [comment]
<p>Here is the related documentation for enabling public and private containers on organizations <a href="https://docs.github.com/en/packages/getting-started-with-github-container-registry/enabling-github-container-registry-for-your-organization">https://docs.github.com/en/packages/getting-started-with-github-container-registry/enabling-github-container-registry-for-your-organization</a></p>
<p>We’re looking into a better way to present this to you when viewing the settings. Thanks for the feedback!</p>

**Comment by dimalvovs** [comment]
nowadays to enable public packages one needs to visit Packages section of organization Settings
<img width="1258" alt="Screenshot 2023-10-19 at 19 14 19" src="https://github.com/community/community/assets/1246862/03f1b8d0-ad78-4404-9cc4-09f5e16cc746">
